src/languages/scala.js

```diff
@@ -117,6 +117,11 @@ export default function(hljs) {
     contains: [ NAME ]
   };
 
+  const EXTENSION = {
+    className: 'keyword',
+    begin: /(?<=^\s*)extension(?=\s+[\[\(])/
```

Comment:
Sorry, we can't use look-behind, Safari still doesn't support this.

Reply:
Consider using a multi-matcher instead and only highlighting the part you care about.

<https://highlightjs.readthedocs.io/en/latest/mode-reference.html#beginscope>

Reply:
Will do this for other PRs as well

Reply:
Done